src/components/map/use-selected-feature-change-effect.tsx

```diff
@@ -73,7 +80,16 @@ const useSelectedFeatureChangeEffect = (
               );
               if (mapLines) {
                 setMapLines(mapLines);
-                fitBounds(fromCoordinates, stopPlaceCoordinates, mapCameraRef);
+                const bbox = getBoundingBox(mapLines);
+                const northEast: Coordinates = {
+                  longitude: bbox.xMin,
+                  latitude: bbox.yMin,
+                };
+                const southWest: Coordinates = {
+                  longitude: bbox.xMax,
+                  latitude: bbox.yMax,
+                };
+                fitBounds(northEast, southWest, mapCameraRef, padding);
```

Comment:
Nice, this seems to be working correctly. I think we should do something like this also when pressing the `My location` button, since now it centres the my location to the middle of the screen, which is pretty much on the border between map and bottom-sheet.

Reply:
Sure, I will do it for other places but perhaps in another task.